salt/metalk8s/kubeadm/init/mark-control-plane/configured.sls

```diff
@@ -0,0 +1,31 @@
+{% set hostname = salt['network.get_hostname']() %}
+
+{% set kubeconfig = "/etc/kubernetes/admin.conf" %}
+{% set context = "kubernetes-admin@kubernetes" %}
+
+Apply control-plane role label:
+  kubernetes.node_label_present:
+    - node: {{ hostname }}
+    - name: "node-role.kubernetes.io/master"
```

Comment:
@TeddyAndrieux prefers `name` to go first, then `kubeconfig` and `context`, then `node` and `value`. I kinda agree (had to be convinced though ;))

Reply:
Done.